demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -810,20 +819,31 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
 
   dataMessageHandler(dataMessage: DataMessage): void {
     if (!dataMessage.throttled) {
+      const isSelf = dataMessage.senderAttendeeId === this.meetingSession.configuration.credentials.attendeeId;
+      if (dataMessage.timestampMs <= this.lastReceivedMessageTimestamp) {
```

Comment:
Did you run into this or is this just to be safe?

Reply:
You can potentially receive old messages when reconnecting; this prevents these messages from being redelivered.